spring/src/main/java/io/seata/spring/annotation/GlobalLock.java

```diff
@@ -64,4 +65,10 @@
      * @return lock retry times
      */
     int lockRetryTimes() default -1;
+
+    /**
+     * @return lock strategy mode
+     */
+    LockStrategyMode lockStrategyMode() default LockStrategyMode.PESSIMISTIC;
```

Comment:
why need it?  GlobalLock only check locks but does not occupy them.

Reply:
> why need it? GlobalLock only check locks but does not occupy them.

The GlobalLock contains some configuration of lock rules